com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -938,6 +938,31 @@ SkinnedMeshRenderer unitySkin
 
                 // add bind pose
                 ExportBindPose (unitySkin, fbxNode, fbxScene, skinnedMeshToBonesMap);
+
+                // now that the skin and bindpose are set, make sure that each of the bones
+                // is set to its original position
+                var bones = unitySkin.bones;
+                foreach (var bone in bones)
+                {
+                    var fbxBone = MapUnityObjectToFbxNode[bone.gameObject];
+                    ExportTransform(bone, fbxBone, newCenter: Vector3.zero, TransformExportType.Local);
+
+                    // Cancel out the pre-rotation from the exported rotation
+
+                    // Get prerotation
+                    var fbxPreRotationEuler = fbxBone.GetPreRotation(FbxNode.EPivotSet.eSourcePivot);
+                    // Convert the prerotation to a Quaternion
+                    var fbxPreRotationQuaternion = ModelExporter.EulerToQuaternion(fbxPreRotationEuler);
```

Comment:
looks like you don't need `ModelExporter.` here since you don't in the other calls